Tests/SwiftParserTest/translated/InitDeinitTests.swift

```diff
@@ -403,4 +393,14 @@ final class InitDeinitTests: XCTestCase {
     )
   }
 
+  func testDeinitInSwiftinterfaceIsFollowedByFinalFunc() {
+    AssertParse(
```

Comment:
Could you check the structure here? This could also be parsed as
```
deinit final
func foo()
```
depending on recovery. Do we already have:
```
class Foo {
  func foo() final
  func bar()
}
```
?